cfgov/cfgov/settings/base.py

```diff
@@ -764,6 +764,8 @@
     # Controls whether or not to include Qualtrics Web Intercept code for the
     # Q42020 Ask CFPB customer satisfaction survey.
     "ASK_SURVEY_INTERCEPT": [],
+    # Hide archive filter options in the filterable UI
+    "ARCHIVE_FILTER_OPTIONS": [("boolean", False)],
```

Comment:
We don't want to force it off in settings. This should be empty. We'll need to manually add the flag in production *before* this code gets there so that it doesn't get turned off unintentionally.
```suggestion
    "ARCHIVE_FILTER_OPTIONS": [],
```

Reply:
Updated so archive filter options is not turned off by default